### DISCUSSION TITLE
【八卦】白祈重工之前发生什么了?

### DISCUSSION THREAD

**Discussion by fly-pigawa** [share121/inter-knot#1564] [其他]
![image](https://github.com/user-attachments/assets/f62219f1-d6f4-4d2f-bb6a-1685031e1a77)
在一个儿童节目上看到了白祈重工的嘉宾，主持人竟然当着嘉宾的面提到白祈重工数年前
有过一个大丑闻。虽然这个主持人明显是竞争对手派来搞事的，可我还是有点在意!
有人知道具体是怎么回事吗?

**Comment by RETRN1** [comment]
一看就是之前社长卷钱跑路咯~

**Comment by lllliufeng** [comment]
好像是旧都陷落前的事吧...远景真是无所不用其极啊...

**Comment by laoyee234** [comment]
好像是主持人当众提起白祈重工前社长圈钱跑路的事，但最近不是洗白了吗（前社长实际上是拿去付尾款来着），请问这位绳匠用的是2g网吗？

**Comment by ZhuYuan-148** [comment]
是当年的社长卷钱一事吧，听说当时对公司造成了不小的影响，现在才勉强走出了低谷。
但实际上白祈重工在新艾利都揽的活不少，很多建筑都是他们建的，质量也不差。

**Comment by blomstqi** [comment]
不太了解，但这个狮子恶意是否太大了些😰

**Comment by 1451374815** [comment]
白祈重工财务出大问题了，感觉跑路吧

**Comment by yys20071108** [comment]
社长卷钱

**Comment by lamtt26** [comment]
把剧情走完就知道了

**Comment by Phaethon-0019** [comment]
这事我持续关注过，据说是因为那个狮子当众说白祇重工前任社长卷钱跑路，说的还挺难听；后来我跟安东和大本闲聊的时候，他们说空洞里那个地方曾经是白祇重工承包的工程，施工的时候挖出来一个不知道是什么的空洞生物，工程就这样被迫搁置。当时白祇重工的空洞用智能重型工程机械还是初代，前社长从白祇重工账上转了一笔钱自己买了一台机器，用自己的命把怪物封在了纪念碑里；他们还给我看了一眼当时前任社长买机器的时候留下的货单，事情才这样水落石出。只能说远景实业上次被曝草菅人命后，这事要想平下去只能看他们公关的水平了
